presto-docs/src/main/sphinx/release/release-0.259.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.259
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a casting error that sometimes occurred for EXPLAIN queries when the query plan contains LIKE predicates.
+* Add Weibull distribution CDF :func:`weibull_cdf` and inverse CDF :func:`inverse_weibull_cdf` functions.
+* Add support to enable verbose memory exceeded errors by default.
+* Add support for CREATE TYPE syntax.
```

Comment:
This needs a link to documentation

Also should be 
```
``CREATE TYPE``
```

Reply:
Looks like the author did not add any documentation for this. 
https://github.com/prestodb/presto/pull/16392
@pranjalssh